convert2rhel/utils.py

```diff
@@ -438,6 +472,12 @@ def download_pkg(
     if set_releasever and system_info.releasever:
         cmd.append("--releasever=%s" % system_info.releasever)
 
+    if not set_releasever and manual_releasever:
+        cmd.append("--releasever=%s" % manual_releasever)
```

Comment:
I see your point. It makes sense. Looking at this now, I would rewrite it as the following: 

```python
if set_releasever:
    if not custom_releasever and not system_info.releasever:
        raise AssertionError("custom_releasever or system_info.releasever must be set.")
        
    if custom_releasever:
        cmd.append("--releasever=%s" % custom_releasever)
    elif system_info.releasever:
        cmd.append("--releasever=%s" % system_info.releasever)
```

This way we require that the `set_releasever` is set to `True` and not the opposite. Since we are dealing with setting a releasever (either custom one or from `system_info`, I would say that `set_releasever` is appropriate here for both cases).

Reply:
Yeah, that looks good. Thanks.